src/python/pants/engine/internals/scheduler.py

```diff
@@ -307,9 +308,9 @@ def visualize_to_dir(self):
     def _metrics(self, session):
         return self._from_value(self._native.lib.scheduler_metrics(self._scheduler, session))
 
-    def poll_workunits(self, session) -> PolledWorkunits:
+    def poll_workunits(self, session, max_verbosity: int) -> PolledWorkunits:
```

Comment:
Maybe have this take `LogLevel` and convert it into `int` here, instead of on line 439.